js/foam/polymer/storage/ArrayBridge.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * @license
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CLASS({
+  package: 'foam.polymer.storage',
+  name: 'ArrayBridge',
+  properties: [
+    'target',
+    'key',
+    'dao',
+    ['feedback_', 0]
+  ],
+  methods: [
+    function onChange(e) {
```

Comment:
Doesn't need to be. It was originally, and needed to be. But it's only called directly by the observer in Polymer-land; it doesn't need the binding.